sass/_omni-search.scss

```diff
@@ -0,0 +1,219 @@
+@import "general-mixins";
+@import "media-queries";
+$form-padding: 0.5rem;
+
+
+@mixin omnisearch-style($version) {
+  @if $version == style-1 {
+    @content; // calls the @mga version here...
```

Comment:
the `@content` is necessary for each of these since the subsequent styles contained  create different visual designs... not sure what the issue is here...